_pages/home/updating-to-version-2211.19/technical-changes-version-2211.19.md

```diff
@@ -6,6 +6,30 @@ The Spartacus migration schematics scan your codebase and inject code comments w
 
 **Note:** If you happen to have in your codebase a custom function or class that has the same name as a function or class that has changed or been removed in the Spartacus public API, there is a chance that the migration script could identify your custom function or class as needing to be updated. In this case, you can ignore and remove the comment.
 
+## Replacing the `<img>` tag with the `<picture>` tag
+
+The media component is now designed to display images using the `<picture>` element. This enables responsive image selection for modern browsers that have support for `srcset` and media queries. The media component falls back to using the `<img>` tag for legacy browsers, when `srcset` is not provided, or when you explicitly declare in `AppModule` to use `<img>` instead of `<picture>`. This approach improves page load performance and user experience by ensuring that the most appropriate image is loaded, based on the viewport or display conditions.
+
+### Configuration
+
+To enable the legacy approach of using `<img>` tags by default, you need to set the `USE_LEGACY_MEDIA_COMPONENT` provider property to `true` in the `app.module.ts`. The following is an example:
+
+```ts
+// code
+```
+
+You can also enable the legacy approach by setting the `useLegacyMediaComponent` property in `MediaConfig` to `true`. The following is an example:
+
+```ts
+// code
+```
+
+In either case, `<img>` will then be used instead of `<picture>`. By default, the `USE_LEGACY_MEDIA_COMPONENT` provider property is set to `false`, and `<picture>` will always be used when the other conditions are met. The fallback property is set to use the `<picture>` element for cases when `USE_LEGACY_MEDIA_COMPONENT` was not provided, or when `useLegacyMediaComponent` was not defined.
```

Comment:
Not sure, but when thinking now, maybe we need to remove all information  about `USE_LEGACY_MEDIA_COMPONENT` token and leave only information that says that when `useLegacyMediaComponent` is not provided in config `<picture>` will always be used when the other conditions are met. By saying other conditions i mean two: browsers support and `srcset` for this current component was provided on the backend side.